test/stripe/api_test.exs

```diff
@@ -16,4 +16,27 @@ defmodule Stripe.APITest do
 
     assert Stripe.APIMock.oauth_request(:post, "www", %{body: "body"}) == :post
   end
+
+  test "oauth_request sets authorization header for deauthorize request" do
+    verify_on_exit!()
```

Comment:
✂️

Reply:
I ran the elixir formatter on this and it keeps the empty line here, but I can remove it if you'd still like to see it gone.

Reply:
Oh I was referring to this mox setup `verify_on_exit!()`

Reply:
Gah! Case of the Mondays or something.